doc/ondemand.md

```diff
@@ -90,8 +105,8 @@ Documents Are Iterators
 A `document` is *not* a fully-parsed JSON value; rather, it is an **iterator** over the JSON text.
 This means that while you iterate an array, or search for a field in an object, it is actually
 walking through the original JSON text, merrily reading commas and colons and brackets to make sure
-you get where you're going. This is the key to On Demand's performance: since it's just an iterator,
-it lets you parse values as you use them. And particularly, it lets you *skip* values you don't want
+you get where you are going. This is the key to On Demand's performance: since it's just an iterator,
```

Comment:
I get why it formal vernacular is good for papers, but maybe not documentation? For me at least, a more informal vernacular and tone in documentation and tutorials make it easier to to read (not at the expense of precision+accuracy, of course). 

At any rate, these changes aren't big enough for me to worry about, so I'm going to merge them.

Reply:
:-) I think we agree that it does not matter very much.